### PR TITLE
Switched to newest Demands

### PR DESCRIPTION
**Pull request by ekoval**
Plus one integrations test fixed.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # [Changelog](https://github.com/yola/sitewit/releases)
 
+## DEV
+
+* Switch to Demands >= 4.0.0
```
Comment (RayeN) [inline CHANGELOG.md]: `== 4.0.0` ?

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by RayeN** [comment]
:+1:

**Pull request merged**